### PR TITLE
Fixes #9032: quality assistant doesn't display test error properly

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9032

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: scripts/rudder-dev/quality-assistant [hunk 1]
```diff
@@ -77,7 +77,8 @@ def pr_merge(repo, url):
     print(stderr+"\n"+output)
   if code == 127:
     # extract 15 first lines of (error then output) to limit comment size
-    msg = "\n".join((stderr+"\n"+output).split()[:15])
+    lines = (stderr+"\n"+output).split('\n')
+    msg = "\n".join(lines[:15]) + "\n[...]\n" + "\n".join(lines[-10:])
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/quality-assistant]: What does lines '-10:' do ?

Comment (peckpeck) [inline scripts/rudder-dev/quality-assistant]: It extracts the last 10 entries

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**